starlark/example_test.go

```diff
@@ -90,7 +90,7 @@ func ExampleThread_Load_sequential() {
 
 			// Load and initialize the module in a new thread.
 			data := fakeFilesystem[module]
-			thread := &starlark.Thread{Load: load}
+			thread := &starlark.Thread{Name: "exec " + module, Load: load}
```

Comment:
There's one more `&starlark.Thread{...}` above. Does that need a `Name`?

Reply:
Done.